src/python/WMCore/MicroService/Unified/MSCore.py

```diff
@@ -95,7 +96,7 @@ def updateTransferInfo(self, requestStatuses):
             doc['transfers'] = records
             self.reqmgrAux.postTransferInfo(wname, doc)
 
-    def getTransferInfo(self, wname):
+    def getTransferInfo(self, wname, since=None):
```

Comment:
Same comment here, move it over to MSMonitor.
In addition to that, provided that the data loaded isn't too much, we should get all those transfer docs in bulk.